### PR TITLE
storyid 176 Added view more on photo gallery section and made a new p…

### PR DESCRIPTION
**Pull request by sadhviadigh**
…age for gallery

Replace last XX with your Clubhouse Story ID: https://app.clubhouse.io/tridentechnologies/story/176

[Brief description of the PR]

- [X] Have you included correct clubhouse story id.?
- [X] Is this branch based on `develop` and the Pull Request is against `develop`? (Or based on `master` and the PR is against `master`)
- [X] Is your story working on your local machine.?
- [X] Is your work tested Locally.?
- [ ] Have You updated the database credentials with matching hosting site.?
- [X] Does this PR contain all related commits to the clubhouse story and only the commits related to that story? (e.g. if your PR have two clubhouse stories you're doing it wrong)
- [X] Have you updated the related CHANGELOG.md?

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -377,7 +375,7 @@ <h3 class="mb-3">Website</h3>
 		<section class="ftco-section ftco-no-pt ftco-no-pb">
 			 <div id="map" class="bg-white"></div>  <center>
 <!-- <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3075.7420291016538!2d74.63407422203885!3d13.767287749930642!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x3bbc8af163ea847d%3A0x8b313b1e7737f447!2sShri%20Agasthyeshwara%20Temple!5e0!3m2!1sen!2sin!4v1588701477117!5m2!1sen!2sin" width="400" height="450" frameborder="0" style="border:0;" allowfullscreen="" aria-hidden="false" tabindex="0"></iframe> </center> -->
-		</section>
+		</section></center>
```
Comment (manjunathpm) [inline index.html:378]: Why have you added </center> tag here.?

File: new 1 [hunk 2]
```diff
@@ -0,0 +1,88 @@
+<div id="ac-wrapper row" style='display:none'>
```
Comment (manjunathpm) [inline new 1:1]: Why do we need this file.?

File: index.html (hunk 1 above)
Comment (sadhviadigh) [reply index.html:378]: I added it mistakenly,  ok i will correct it in my pc and then  i will push..

File: new 1 (hunk 2 above)
Comment (sadhviadigh) [reply new 1]: yhe we don't need that , i will remove that even

**Pull request merged**